AsyncDisplayKitTests/ASTableViewTests.m

```diff
@@ -155,28 +155,34 @@ - (void)testReloadData
   [tableView reloadData];
   
   for (int i = 0; i < NumberOfReloadIterations; ++i) {
-    UITableViewRowAnimation rowAnimation = (arc4random_uniform(1) == 0 ? UITableViewRowAnimationMiddle : UITableViewRowAnimationNone);
-    
-    BOOL animatedScroll               = (arc4random_uniform(1) == 0 ? YES : NO);
-    BOOL reloadRowsInsteadOfSections  = (arc4random_uniform(1) == 0 ? YES : NO);
-    BOOL letRunloopProceed            = (arc4random_uniform(1) == 0 ? YES : NO);
-    BOOL useBeginEndUpdates           = (arc4random_uniform(2) == 0 ? YES : NO);
+    UITableViewRowAnimation rowAnimation = (arc4random_uniform(2) == 0 ? UITableViewRowAnimationMiddle : UITableViewRowAnimationNone);
+    BOOL animatedScroll               = (arc4random_uniform(2) == 0 ? YES : NO);
```

Comment:
arc4random_uniform(upper_bound) returns a value between 0..upper_bound-1, so arc4random_uniform(1) === 0, which means these were all previously YES. Additionally, the runLoopDelay calculation wasn't using floating point so it was always returning either 0 or (very rarely) 1. So it just wasn't very random, although it looked like it was.